### PR TITLE
Add script for pytest coverage test

### PR DESCRIPTION
**Pull request by RamosFe**
|Related issue|
|---|
| https://github.com/wazuh/wazuh/issues/26466 |

## Description

Adds a script and the dependencies to get the Python unit tests coverage.

### CODE REVIEW

**Review by GGP1 — APPROVED** [review]
LGTM

File: requirements.txt
Comment (fdalmaup) [inline requirements.txt:1]: We need to check that the `jsonschema` and `pytest` upgrades do not break other tests that may use them. Please, run in https://github.com/wazuh/wazuh/pull/26470 any related test for verification.

**Review by fdalmaup — CHANGES_REQUESTED** [review]

File: requirements.txt
Comment (RamosFe) [reply requirements.txt:1]: Sent to run the test that uses the library. When the tests are finished, I will add the results to the mentioned PR.

**Comment by RamosFe** [comment]
## Update - Tests

I ran all the tests that use the `qa-integration-framework` as well as the `jsonschemma` and `pytest` dependencies:

> All tests ran without any errors. The UT tests were run in the following PR https://github.com/wazuh/wazuh/pull/26470.

- **Syscollector Windows**: https://github.com/wazuh/wazuh/actions/runs/11710112533/job/32615899592
- **Syscollector Linux**: https://github.com/wazuh/wazuh/actions/runs/11710115470
- **Remoted**: https://github.com/wazuh/wazuh/actions/runs/11710151009/job/32615787491
- **Office365**: https://github.com/wazuh/wazuh/actions/runs/11710164964/job/32615832141
- **IT API Tier 0-1**: https://github.com/wazuh/wazuh/actions/runs/11710197723
- **IT API Tier 2**: https://github.com/wazuh/wazuh/actions/runs/11710213368
- **Authd**: https://github.com/wazuh/wazuh/actions/runs/11710289059/job/32616244603
- **AWS**: https://github.com/wazuh/wazuh/actions/runs/11710294963/job/32616263277
- **Agentd Linux**: https://github.com/wazuh/wazuh/actions/runs/11710356528
- **Agentd Windows**: https://github.com/wazuh/wazuh/actions/runs/11710362921/job/32616691717
- **Analysisd**: https://github.com/wazuh/wazuh/actions/runs/11710367391
- **Execd Linux**: https://github.com/wazuh/wazuh/actions/runs/11710375963/job/32616520020
- **Execd Windows**: https://github.com/wazuh/wazuh/actions/runs/11710377429
- **Enrollment Windows**: https://github.com/wazuh/wazuh/actions/runs/11710383151
- **Enrollment Linux**: https://github.com/wazuh/wazuh/actions/runs/11710305193
- **RBAC (Framework)**: https://github.com/wazuh/wazuh/actions/runs/11710390443
- **MsGraph**: https://github.com/wazuh/wazuh/actions/runs/11710392830
- **SCA Linux**: https://github.com/wazuh/wazuh/actions/runs/11710394375
- **SCA Windows**: https://github.com/wazuh/wazuh/actions/runs/11710395777
- **WazuhDB**: https://github.com/wazuh/wazuh/actions/runs/11710408274
- **Github Linux**: https://github.com/wazuh/wazuh/actions/runs/11710416059
- **Github Windows**: https://github.com/wazuh/wazuh/actions/runs/11710417211

**Review by fdalmaup — APPROVED** [review]
LGTM

**Pull request merged**